app/models/operatingsystem.rb

```diff
@@ -238,6 +238,11 @@ def deduce_family
     end
   end
 
+  def self.authorized_resource_name
+    # We don't want STI subclasses to have separate permissions
+    Operatingsystem.name
```

Comment:
Because if the name of the constant changes, it will raise error right away (unlike with the string, where you have to have tests for that or hope that you will notice that). Also, refactoring tools work with this better (not that I personally use any IDE:)